redis.go

```diff
@@ -105,6 +119,10 @@ type Options struct {
 	// than specified in this option.
 	// Default: 0 = no eviction
 	IdleTimeout time.Duration
+
+	// MaxRetries specifies maximum number of times client will retry
+	// failed command. Default is to not retry failed command.
+	MaxRetries int
```

Comment:
MaxRetries and MaxRedirects are two different things really. We would e.g. never allow to set `MaxRetries` on ClusterOptions, right?